packages/framework/src/Resources/views/Front/Form/deliveryAddressChoiceFields.html.twig

```diff
@@ -0,0 +1,52 @@
+{% block delivery_address_choice_row %}
+    {% if deliveryAddresses|length > 0 %}
+        <div class="wrap-divider">
+            <table class="table-main">
+                <thead>
+                    <tr>
+                        <th>&nbsp;</th>
+                        <th>{{ 'First name'|trans }}</th>
+                        <th>{{ 'Last name'|trans }}</th>
+                        <th>{{ 'Company name'|trans }}</th>
+                        <th>{{ 'Phone'|trans }}</th>
+                        <th>{{ 'Street'|trans }}</th>
+                        <th>{{ 'City'|trans }}</th>
+                        <th>{{ 'Postcode'|trans }}</th>
+                        <th>{{ 'Country'|trans }}</th>
+                        <th>&nbsp;</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for child in form %}
+                        {% if child.vars.value|isInteger %}
+                            {% set deliveryAddress = deliveryAddresses[child.vars.value] %}
+                            <tr class="js-delivery-address-row">
+                                <td>{{ form_widget(child, { attr: { class: 'js-delivery-address-input' } }) }}</td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.firstName }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.lastName }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.companyName }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.telephone }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.street }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.city }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.postcode }}</label></td>
+                                <td><label for="{{ child.vars.id }}">{{ deliveryAddress.country.name }}</label></td>
+                                <td>
+                                    <a data-href="{{ url('front_customer_delivery_address_delete') }}" class="js-delivery-address-remove-button">
+                                        <i class="svg svg-remove"></i>
+                                    </a>
+                                </td>
+                            </tr>
+                        {% else %}
+                            <tfoot>
+                                <tr>
+                                    <td>{{ form_widget(child, { attr: { class: 'js-delivery-address-input' } }) }}</td>
+                                    <td colspan="9"><label for="{{ child.vars.id }}">{{ 'New shipping address'|trans }}</label></td>
```

Comment:
you are sometimies calling it shipping and sometimes delivery address so please change it everywhere to delivery address

Reply:
Well displayed name in administration and frontend is Shipping address, but in code it is always called delivery address... I know... confusing, but that is how it is

Reply:
so can you please change it to shipping address in all places? you have it as delivery adderess in eg. deliveryAddressListFields.html.twig

Reply:
I have united the naming

Reply:
Also I found, that better is to use delivery address as described in commit message:

https://www.proz.com/kudoz/english/business-commerce-general/2766359-shipping-address-and-delivery-address.html
Explanation:
shipping address implies the port where the consignment is unloaded (e.g. FOB Hamburg) whereas delivery address refers to the place the consignee or addressee is resident in. The goods are unloaded in Hamburg from where they will be dispatched to the delivery address.